### DISCUSSION TITLE
Woo Mollie support for Cart and Checkout Blocks.

### DISCUSSION THREAD

**Discussion by ghost** [woocommerce/woocommerce#45084] [WooCommerce, Blocks, and Site Editing]
## What do you think is missing from the Cart & Checkout blocks?

Other payment gateways.
The payment gateways offered by the official Woo Mollie plugin (in the Netherlands) are not visible in the C&C block as enabled payment methods.

**Comment by nerrad** [comment]
Hi there @EgbertJan  👋 

Thanks for taking the time to make us aware that the Woo mollie plugin isn't working with the Checkout and Cart blocks. These blocks are still in early access (via the feature plugin) and it's _expected_ that some existing Woo Extensions that integrate with the current checkout flow in WooCommerce core will not work with the blocks. 

Currently it's up to payment method extension developers to add support for the new checkout flow offered via the cart and checkout blocks. We have some [documentation on that here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/extensibility/payment-method-integration.md) and will be publicising this more in the coming months. 

Your best course of action right now if you are using a payment method that isn't working with the blocks is to notify the developer of the extension and ask if they can add support.

Until then you can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/).
